sotopia/generation_utils/generate.py

```diff
@@ -291,10 +316,10 @@ def obtain_chain(
     Using langchain to sample profiles for participants
     """
     model_name = _return_fixed_model_version(model_name)
-    chat = ChatLiteLLM(
+    chat = PatchedChatLiteLLM(
         model=model_name,
         temperature=temperature,
-        max_tokens=2700,  # tweak as needed
+        # max_tokens=2700,  # tweak as needed
```

Comment:
may need to specify this programmatically for some providers/models, can be done case-by-case as mentioned in the pr

Reply:
As for now, this is still controlled by LLM_Name. As we deprecate it through #43, we will need a new mechanism to verify the max_tokens.